examples/offline_inference/torchrun_example.py

```diff
@@ -0,0 +1,68 @@
+# see https://github.com/vllm-project/vllm/issues/11400 for
+# the motivation and use case for this example.
```

Comment:
Should also include the instruction about how to run this script (i.e., `torchrun`).

Reply:
fixed in [7f9879a](https://github.com/vllm-project/vllm/pull/12071/commits/7f9879a63243f65a5e30bd7d672ecf0194877e8a)